tests/kvasir-tests/Makefile.common

```diff
@@ -56,13 +57,15 @@ ifeq (washington,$(findstring washington,$(shell uname -n)))
 	GOAL := goal-CentOS-UWCSE
 else
 # Standard version of CentOS
-	GOAL := goal-CentOS
+       GOAL := goal-CentOS-Std
```

Comment:
I don't see any `.goal-CentOS-Std` files in the repository.  Why can't this remain just `.goal-CentOS`?

Reply:
There are a couple as part of this change.  They represent goal files that are unique to the standard version of CentOS (as we use on Travis).  With that change, we can use .goal-CentOS for goal files common to both the Travis and UWCSE versions of CentOS - of which there are several.  Similar to Fedora25 vs. Fedora.

Reply:
I find the mention of a "standard" version of CentOS confusing.  Every version of CentOS  has a version number.  Shouldn't this be just `goal-CentOS` to act as the default?  Both a `.goal-CentOS` and a `.goal-CentOS-UWCSE` can exist, and I think that's clearer than the name "Std" that would require a definition.

Reply:
The intent is to have a 'common' CentOS goal file for when CentOS and CentOS-UWCSE are identical.  (Like Fedora for Fedora24==Fedora25.)  Some name is then needed for when goal-CentOS-UWCSE==goal, but the non-UW version of CentOS is different.  So there are two choices: go back to having duplicate files or choose a different name for CentOS-Std.

Reply:
> Some name is then needed for when goal-CentOS-UWCSE==goal, but the non-UW version of CentOS is different.

Thanks for the explanation.  This makes sense now.
I would call "the non-UW version of Centos" something more like CentOS-nonUWCSE rather than Centos-Std.
Thanks!